llvm/lib/Target/LoongArch/LoongArchISelLowering.cpp

```diff
@@ -141,7 +141,8 @@ LoongArchTargetLowering::LoongArchTargetLowering(const TargetMachine &TM,
 
     setOperationAction(ISD::BITREVERSE, MVT::i32, Custom);
     setOperationAction(ISD::BSWAP, MVT::i32, Custom);
-    setOperationAction({ISD::UDIV, ISD::UREM}, MVT::i32, Custom);
+    setOperationAction({ISD::SDIV, ISD::UDIV, ISD::SREM, ISD::UREM}, MVT::i32,
```

Comment:
Is this a relevant change? Or can it be split into a separate PR?

Reply:
This PR is related with #92205, except for the part of div32
A separate PR may be a good choice.